src/marketing/recommendation-incl-excl.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Control Displayed Recommendations
+tag: product-recommendations
+ee_only: true
+---
+
+When you [create a product recommendation unit]({% link marketing/create-new-rec.md %}), you can apply rules that control which products are displayed in recommendations. These rules are based on a set of inclusion or exclusion conditions. Only products matching all inclusion conditions are eligible to be recommended. Products matching any of the exclusion conditions are not recommended.
+
+Inclusion and exclusion conditions can be static or dynamic. A static condition uses existing product attributes to determine which products are eligible to appear in the unit. For example, you can specify that you only want products that are greater than $25 and are not on sale to appear in the unit. A dynamic condition uses a set of dynamic attributes to determine which products are eligible to appear in the unit. For example, most apparel has an attribute of `gender` with a value of `men` or `women`. If the shopper is browsing men's jackets, a dynamic condition only recommends products where `gender = men`. Static conditions are available on all page types. Dynamic conditions are available on a subset of page types. See the [static versus dynamic conditions](#staticdyn) table to learn more.
+
+{:.bs-callout-info}
+Enabling inclusion and exclusion conditions requires that you have version 3.11.0 or later of the `magento/product-recommendations` module. See the [release notes](https://devdocs.magento.com/recommendations/release-notes.html) to learn more.
+
+## Types of conditions
+
+Magento provides the following inclusion and exclusion conditions you can use to control which products are displayed in recommendations.
+
+- **Category** - Filters based on a product's category. Magento uses direct category assignments and their subcategories. For example, if the category of the product is `Gear`, enabling this rule includes or excludes products that have categories under `Gear`, such as `Gear/Bags` or `Gear/Fitness Equipment`.
+- **Price** - Filters based on a product's price. Magento uses a product's final price when performing the comparison. The final price includes any discounts or special pricing applied.
+- **Product** - Specifies which products are eligible or not eligible to be displayed.
```

Comment:
```suggestion
- **Product** - Specifies which specific products are eligible or not eligible to be displayed in recommendations. You cannot select products that are disabled or not visible individually because those products can never appear in recommendations.
```